pkg/input/readdir.go

```diff
@@ -127,7 +127,10 @@ func ReadDir(fsys fs.FS, cfg config.Application, cfgFilePath string) (*core.Inpu
 			// (so './dist/index.js' becomes 'index.js')
 			relPath = info.Name()
 		}
-		var f core.File = &core.FileRef{FPath: relPath}
+		var f core.File = &core.FileRef{
+			FPath:          relPath,
+			RootConfigPath: cfg.Path,
```

Comment:
One thing to be mindful of here is that this path is relative to the "root", which is split from the original passed-in path in `splitPathRoot`. This is to prevent plugins from accessing arbitrary files (eg, `FPath: "../../../etc/passwd"`).

Reply:
when do we allow access to files outside of the root of the klotho run? Im failing to understand this a little bit so do you have a scenario?

Reply:
This was more a defense against mistakes and third-party plugins (when we get them). Might not be necessary for now, but it does mean this could be incorrect - try doing `klotho ../ --app blah`

Reply:
makes sense, let me make a note of that and run through that scenario, if it doesnt work, ill cut a card to fix it